doc/system_health_monitoring/system-health-HLD.md

```diff
@@ -148,8 +148,15 @@ Considering that different vendors platform may have different LED color capabil
 }
 ```
 
-The field "booting" is deprecated because there is no booting stage anymore. For backward compatible, user can still configure this field but it won't take effect.
+If not configuration is provided, default configuration is used:
 
+```python
+DEFAULT_LED_CONFIG = {
+	'fault': 'red',
+	'normal': 'green',
+	'booting': 'red'
```

Comment:
can we have some other color other than 'red' which is already used for indicating 'fault'?

Reply:
Before this PR, there is no booting stage. So, only two cases:
1. System is all good, LED is green
2. System is not all good, LED is red

Booting stage is introduced by this PR. So, setting "red" as the default value for booting stage is for backward compatible. User can override this value by using a system health configuration file.